autoload/goyo.vim

```diff
@@ -33,8 +33,12 @@ function! s:get_color(group, attr)
 endfunction
 
 function! s:set_color(group, attr, color)
-  let gui = a:color =~ '^#'
-  execute printf('hi %s %s%s=%s', a:group, gui ? 'gui' : 'cterm', a:attr, a:color)
+  if empty(a:attr)
```

Comment:
I see. However, `let gui = a:color =~ '^#'` is still clearly wrong, so this is what I think we should do.

```vim
function! s:set_color(group, attr, color)
  if a:color == 'NONE' || empty(a:attr)
    execute printf('hi %s cterm%s=%s gui%s=%s', a:group, a:attr, a:color, a:attr, a:color)
  else
    let gui = a:color =~ '^#'
    execute printf('hi %s %s%s=%s', a:group, gui ? 'gui' : 'cterm', a:attr, a:color)
  endif
endfunction
```